internal/repository/get_task_from_search.go

```diff
@@ -0,0 +1,29 @@
+package repository
+
+import (
+	"database/sql"
+	"time"
+)
+
+func (r *Repository) GetTaskFromSearch(search string) (*sql.Rows, error) {
```

Comment:
Плохо, что тянешь внутреннюю сущность наружу, тут лучше возвращать свой тип
```suggestion
func (r *Repository) GetTaskFromSearch(search string) ([]config.Task, error) {
```